### PR TITLE
Hotfix sprint09 tijn fix form validation on clone

### PR DESCRIPTION
**Pull request by tschmits**

### CODE REVIEW

File: app/scripts/components/scenariobuilder.js [hunk 1]
```diff
@@ -198,8 +191,13 @@ var exports = (function() {
             // set the template, somehow a computed setter was not working...
             this.selectTemplate(template);
 
+            // set the dataloaded to true, such that the <form> DOM will be rendered
             this.dataLoaded = true;
 
+            // Initialize the tooltips: We do this after the DOM update.
+            this.$nextTick(function () {
+              this.updateAfterTick();
+            });
```
Comment (tschmits) [inline app/scripts/components/scenariobuilder.js]: This has been moved from r.227: this location guarantees that the tagsinput plugin is initialized at the right time, even when cloning.

File: gulpfile.babel.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const reload = browserSync.reload;
 var args = require("yargs").argv;
 // Server used for serving remote url"s
 // "http://136.231.10.175:8888";
-var apiServer = "http://delft3dgt-django:8000";
+var apiServer = "https://delft3dgt-nginx";
```
Comment (tschmits) [inline gulpfile.babel.js]: This reverts the previous change, which was not supposed to be committed in the first place.

This change has no influence on the GUI in production.

File: app/templates/model-details.html [hunk 3]
```diff
@@ -239,22 +239,22 @@ <h2 class="panel-title">{{activeModel.data.name}}</h2>
             </p>
 
             <div class="form-group input-group" @click.stop.prevent="toggle('export_d3dinput', true)">
-              <span class="input-group-addon"><input type="checkbox" class="downloadoption" value="export_d3dinput" v-model="selectedDownloads.export_d3dinput" @click.stop="return false;"></span>
+              <span class="input-group-addon"><input type="checkbox" class="downloadoption" value="export_d3dinput" v-model="selectedDownloads.export_d3dinput" @click.stop="doNothing"></span>
```
Comment (SiggyF) [inline app/templates/model-details.html]: Perhaps include 
`<!-- @click.stop="doNothing" captures an event and avoids bubbling it down, so we don't get 2 toggles -->`

File: app/templates/scenario-builder.html [hunk 4]
```diff
@@ -28,7 +28,7 @@
           </div>
 
           <!-- if user selected something, show GUI -->
-          <div>
+          <div v-if="template && dataLoaded">
```
Comment (SiggyF) [inline app/templates/scenario-builder.html]: There is also an option for an[ async components](http://v1.vuejs.org/guide/components.html#Async-Components)  if you need to load data at component instantiation .

Comment (tschmits) [inline app/templates/scenario-builder.html]: I'll add this as a comment. At the moment, so close to delivery, I would like to refrain from rewriting the component on that level.

**Pull request merged**